src/Nethermind/Nethermind.KeyStore/Config/KeystoreConfig.cs

```diff
@@ -32,5 +34,6 @@ public class KeyStoreConfig : IKeyStoreConfig
         public int SymmetricEncrypterKeySize { get; set; } = 128;
         public int IVSize { get; set; } = 16;
         public string TestNodeKey { get; set; }
+        public string BlockAuthorAccount { get; set; }
```

Comment:
I was thinking of something similar but moving it to something like 'MiningConfig' with 'ForceSealing'